medusa/medusacli.py

```diff
@@ -58,9 +58,14 @@ def configure_logging(verbosity, without_log_timestamp):
     else:
         log_format = '[%(asctime)s] %(levelname)s: %(message)s'
 
+    logging_handlers = [logging.StreamHandler()]
+    if log_file:
+        logging_handlers.append(logging.FileHandler(log_file))
```

Comment:
Could we make this a `RotatingFileHandler` instead? It's easy to forget about the logs until they fill the disks ;)

Reply:
http://www.blog.pythonlibrary.org/2014/02/11/python-how-to-create-rotating-logs/

Reply:
Done!